### ISSUE TITLE
setUsePredictedOrientation() not used

### ISSUE THREAD

**Issue by kylemcdonald** [obviousjim/ofxOculusDK2#10]
The value passed to setUsePredictedOrientation() is not used anywhere. There seems to be a line at the beginning of getOrientationQuat() that would account for this, or possibly the `ovr_GetTimeInSeconds()` call should be replaced with a `0` when prediction is disabled?

**Comment by obviousjim** [comment]
Yeah try it :)

I think this got lost in the DK1 to dk2 shuffle

On Saturday, March 7, 2015, Kyle McDonald notifications@github.com wrote:

> The value passed to setUsePredictedOrientation() is not used anywhere.
> There seems to be a line at the beginning of getOrientationQuat() that
> would account for this, or possibly the ovr_GetTimeInSeconds() call
> should be replaced with a 0 when prediction is disabled?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/obviousjim/ofxOculusDK2/issues/10.

## 
- James

**Comment by kylemcdonald** [comment]
thanks @obviousjim! if you want help with this repo let me know and i can merge things that are working for me and get master up to date as i figure out what's going on.

**Comment by obviousjim** [comment]
Yes please pull request as you go! there are a few of us working on this so
I'll try to get everyone in touch so we can move in step.

On Saturday, March 7, 2015, Kyle McDonald notifications@github.com wrote:

> thanks @obviousjim https://github.com/obviousjim! if you want help with
> this repo let me know and i can merge things that are working for me and
> get master up to date as i figure out what's going on.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/obviousjim/ofxOculusDK2/issues/10#issuecomment-77699903
> .

## 
- James
